rplugin/python3/denite/source/menu.py

```diff
@@ -0,0 +1,59 @@
+# ============================================================================
+# FILE: line.py
+# AUTHOR: Shougo Matsushita <Shougo.Matsu at gmail.com>
+#         TJ DeVries <devries.timothyj at gmail.com>
+# License: MIT license
+# ============================================================================
+
+from .base import Base
+
+
+class Source(Base):
+
+    def __init__(self, vim):
+        Base.__init__(self, vim)
+
+        self.name = 'menu'
+        self.kind = 'jump_list'
+
+        # self.matchers = []
+        # self.sorters = []
+
+    def on_init(self, context):
+        # TODO: Could also check for unite_source_menu_menus?
+        context['__menus'] = self.vim.vars['denite_source_menu_menus']
+
+    def gather_candidates(self, context):
+        lines = []
+
+        menu_context = [option for option in context['sources'] if option['name'] == self.name][0]
```

Comment:
This change didn't work for me. I left it the way it is for now. I am getting the file candidates to pop up with the original way I had it, but not with the other way.